README.md

```diff
@@ -132,27 +178,10 @@ We encourage some combination of OOCSS and BEM for these reasons:
   * CSS Trick's [BEM 101](https://css-tricks.com/bem-101/)
   * Harry Roberts' [introduction to BEM](http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/)
 
-We recommend a variant of BEM with PascalCased “blocks”, which works particularly well when combined with components (e.g. React). Underscores and dashes are still used for modifiers and children.
+We recommend a variant of BEM with PascalCased “blocks”.
```

Comment:
Not sure which conventions this specifically refers to, but I think we should continue using our existing conventions for modifiers and children.
See https://gist.github.com/jico/f9cb67634387b9eb3591#componentname--modifiername
and https://gist.github.com/jico/f9cb67634387b9eb3591#componentname-descendantname

These conventions have been working out very well in existing applications and are used in many of our Crowdtap, Carbon, and Angular styles.